libs/core/tests/unit_tests/messages/test_imports.py

```diff
@@ -1,6 +1,8 @@
 from langchain_core.messages import __all__
 
 EXPECTED_ALL = [
+    "MessageLikeRepresentation",
+    "_message_from_dict",
```

Comment:
this one is private -- could we remove it?

Reply:
See the previous comment